backend/core/migrations/0005_email_unique_in_db.py

```diff
@@ -0,0 +1,62 @@
+import logging
+from django.db import migrations, models
```

Comment:
Very minor coding style suggestion.

As recommended by [PEP8](https://www.python.org/dev/peps/pep-0008/#imports) and the Django [coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#imports), we should use a blank line between imports related to the Python standard library and the `django` module.

```suggestion

from django.db import migrations, models
```